mantidproject/mantid
# share normalization between runs in muon analysis

~~To be merged after #19706~~

The normalization can now be shared between runs in TF asymmetry mode of muon analysis. 
**To test:**


To test load some muon data (e.g. MUSR00064748 ) and go to the data analysis tab. The normalization should be shown in the data table. The row below (fix normalization) should be false. 

If you add a function to fit to. Then look in the fit menu, TF Asymmetry fit should be enabled. 

If you then click the tick box for fix normalization (so its true) then look in the fit menu, TF asymmetry fit should be disabled. Ticking the box again should enable the TF asymmetry fit. 

Make a note of your normalization. Make sure that keep normalization is ticked. Go to the home tab and load some different data (e.g. MUSR00064728). Then go to the data analysis tab. The normalization should be the same as the one you noted down. 

The plot will probably not be centered on zero. As a final check untick the keep normalization box and then reload the data (you will have to load something else then reload the data). The normalization should be different and the plot centered on zero. 

Fixes #19715 



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
